sklearn/ensemble/tests/test_bagging.py

```diff
@@ -589,28 +589,6 @@ def test_bagging_with_pipeline():
     assert isinstance(estimator[0].steps[-1][1].random_state, int)
 
 
-class DummyZeroEstimator(BaseEstimator):
-    def fit(self, X, y):
-        self.classes_ = np.unique(y)
-        return self
-
-    def predict(self, X):
-        return self.classes_[np.zeros(X.shape[0], dtype=int)]
-
-
-def test_bagging_sample_weight_unsupported_but_passed():
-    estimator = BaggingClassifier(DummyZeroEstimator())
-    rng = check_random_state(0)
-
-    estimator.fit(iris.data, iris.target).predict(iris.data)
-    with pytest.raises(ValueError):
-        estimator.fit(
-            iris.data,
-            iris.target,
-            sample_weight=rng.randint(10, size=(iris.data.shape[0])),
-        )
```

Comment:
This test is no longer needed I think. It is now okay to have a base estimator that does not support `sample_weight`.